controllers/constants/caikit-metrics.json

```diff
@@ -0,0 +1,55 @@
+{
+    "metrics": {
+        "supported": "true",
+        "config": [
+            {
+                "title": "Number of requests",
+                "type": "REQUEST_COUNT",
+                "queries": [
+                    {
+                        "title": "Number of successful incoming requests",
+                        "query": "sum(increase(predict_rpc_count_total{namespace='${NAMESPACE}',code='OK',model_id='${MODEL_ID}'}[${RATE_INTERVAL}]))"
```

Comment:
I missed this earlier, but can we change `${MODEL_ID}` to `${MODEL_NAME}`? The replacement logic is written for `${MODEL_NAME}`